src/ceph-disk

```diff
@@ -1719,6 +1804,9 @@ def main_prepare(args):
 
         if args.zap_disk is not None:
             zap(args.data)
```

Comment:
I have not checked that this issue exists on any platform other than SUSE SLE 12.